content/en/docs/concepts/storage/projected-volumes.md

```diff
@@ -71,18 +71,29 @@ A container using a projected volume source as a [`subPath`](/docs/concepts/stor
 volume mount will not receive updates for those volume sources.
 {{< /note >}}
 
-## SecurityContext interactions
+### serviceAccountToken and securityContext
 
 The [proposal](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#proposal) for file permission handling in projected service account volume enhancement introduced the projected files having the the correct owner permissions set.
 
-### Linux
+#### Linux
 
-In Linux pods that have a projected volume and `RunAsUser` set in the Pod
-[`SecurityContext`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context),
-the projected files have the correct ownership set including container user
-ownership.
+In some cases, Kubernetes applies a security optimization for the contents of `serviceAccountToken`
+volumes.
```

Comment:
I'll defer to docs reviewers on the phrasing, but I think it's slightly clearer to drop this sentence and just lead with "When all containers in a pod have the same `runAsUser`..."

Otherwise, I'd get distracted by the "in some cases" and start wondering what those cases were and whether the one listed below is exhaustive